docs/reference/query-languages/esql/_snippets/functions/layout/match_phrase.md

```diff
@@ -2,8 +2,7 @@
 
 ## `MATCH_PHRASE` [esql-match_phrase]
 ```{applies_to}
-stack: preview 9.1.0
-serverless: preview
+stack: unavailable 9.0, ga 9.1.0
```

Comment:
@leemthompo I don't think we want to add an explicit `unavailable` lifecycle state for states before a feature was available